egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -188,37 +189,42 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
             scp:"cat ${data_feats}/${train_set}/feats.scp ${data_feats}/${dev_set}/feats.scp |" \
             "${data_feats}/${train_set}/cmvn.npy"
     fi
-
 fi
 
 
+token_list="data/token_list/${trans_type}/tokens.txt"
 if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
-    log "stage 3: Generate character level token_list from ${srctexts}"
+    log "Stage 3: Generate character level token_list from ${srctexts}"
     mkdir -p "$(dirname ${token_list})"
-
+    # "nlsyms_txt" should be generated by local/data.sh if need
+    if [ -n "${nlsyms_txt}" ]; then
+        nlsyms="$(<${nlsyms_txt})"
+    else
+        nlsyms=
+    fi
     echo "<unk>" > "${token_list}"
     if [ -n "${nlsyms}" ]; then
-        cat ${srctexts} | pyscripts/text/text2token.py -s 1 -n 1 -l "${nlsyms}"  \
+        pyscripts/text/text2token.py -s 1 -n 1 -l "${nlsyms}" < "${srctexts}" \
```

Comment:
I expected that `srctexts` can be multiple texts, like srctexts="a.txt b.txt", but I don't know how to solve this with shell check.

Reply:
I see. I wil revert it.